src/pybind/ceph_argparse.py

```diff
@@ -924,12 +924,12 @@ def validate(args, signature, flags=0, partial=False):
 
             # no arg, but not required?  Continue consuming mysig
             # in case there are later required args
-            if myarg == None and not desc.req:
+            if not myarg and not desc.req:
```

Comment:
how about
```py
if myarg in (None, []) and not desc.req:
```
?

otherwise this change breaks the feature #15282 wanted to introduce. because if `a` is an empty string, we take it as an invalid param.